service/src/main/resources/changelog/changesets/20230207_sarscov2_nextstrain_nonempty_optionals.yaml

```diff
@@ -0,0 +1,1313 @@
+databaseChangeLog:
+  - changeSet:
+      id: "update sarscov2_nextstrain: remove non_empty field from optionals"
+      author: mspector
+      changes:
+        - update:
+            tableName: run_set
+            where: run_set_id='10000000-0000-0000-0000-000000000006'
+            columns:
+              - column:
+                  name: input_definition
+                  value: |
+                    [
+                      {
+                        "input_name": "sarscov2_nextstrain.build_name",
+                        "input_type": {
+                          "type": "primitive",
+                          "primitive_type": "String"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "build_name"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.builds_yaml",
+                        "input_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "build_yaml"
+                        }
+                      },
+                      {
+                        "input_name": "sarscov2_nextstrain.assembly_fastas",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "array",
+                            "array_type": {
+                              "type": "optional",
+                              "optional_type": {
+                                "type": "primitive",
+                                "primitive_type": "File"
+                              }
+                            },
+                            "non_empty": false
+                          }
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "combined_assemblies" 
```

Comment:
this was the other change. I was told that the `assembly_fastas` input variable defaults to a Google Cloud file, so we need to set this to read from the `combined_assemblies` data table attribute instead.